napari/_qt/qt_viewer.py

```diff
@@ -194,9 +197,10 @@ def __init__(self, viewer: Viewer, show_welcome_screen: bool = False):
             'napari:toggle_console_visibility',
             self.viewerButtons.consoleButton,
         )
-        action_manager.bind_shortcut(
-            'napari:toggle_console_visibility', 'Control-Shift-C'
-        )
+
+        for action, shortcuts in SETTINGS.shortcuts.shortcuts.items():
```

Comment:
Let's move this to a method, but first unbind and then bind
```python
action_manager.unbind_shortcut(action\)
action_manager.bind_shortcut(action, shortcut)
```

Reply:
@Carreau question, if we unbind an action, will it unbind ALL the shortcuts previously bound?

Reply:
Another question regarding this: Can you bind more than 1 shortcut in the action manager?  It looks like in the method for binding a shortcut, it will get overwritten with more than 1.  When I looked at the shortcuts saved for each action, that seemed to be the case. 
```
def bind_shortcut(self, name: str, shortcut: str) -> None:
        """
        bind shortcut `shortcut` to trigger action `name`

        Parameters
        ----------
        name : str
            name of the corresponding action in the form ``packagename:name``
        shortcut : str
            Shortcut to assign to this action use dash as separator. See
            `Shortcut` for known modifiers.

        Notes
        -----
        calling `bind_button` can be done before an action with the
        corresponding name is registered, in which case the effect will be
        delayed until the corresponding action is registered.
        """
        self._validate_action_name(name)
        self._shortcuts[name] = shortcut
        self._update_shortcut_bindings(name)
        self._update_gui_elements(name)
```

Reply:
The action manager currently only support a single shortcut per action, but we could put several that would not be a problem. That's one of the question that need to be resolved as to whether we _want_ to do that; as well as wether we want the shortcut manager to also support the generator-based shortcut that trigger an action on key press and release.